docs/wiki/configurator/gps-tab.md

```diff
@@ -4,8 +4,52 @@ sidebar_position: 12
 
 # GPS Tab
 
-:::note
-Sorry; this Wiki page has not yet been written.
-:::
+Set up your GPS settings here. GPS is used in Betaflight for GPS Rescue and showing GPS Coordinates in the OSD.
+
+![GPS tab](/img/betaflight_configurator_gps_tab.png)
+
+## GPS Configuration
+
+Here’s the revised table with the protocol options combined:
+
+| **Option**                   | **Description**                                                                                                                                                                                                                                        |
```

Comment:
Do not think we need to use bold text in tables for headers and first column. But did not lookup rules on that :)

Reply:
Maybe.
I will fix this and make styling more consistent tomorrow over the next day.

Reply:
Leaving this here (as removing from my PR

```
# GPS

The correct term to use is GNSS.

GPS is just one of the available Global navigation satellite systems.

## Systems

| Constellation | Operator       | Coverage | Altitude (km)                        | Satellites in Orbit |
| ------------- | -------------- | -------- | ------------------------------------ | ------------------- |
| BEIDOU        | CNSA           | Global   | 21,528 (MEO)<br/>35,768 (GEO & IGSO) | 48                  |
| GALILEO       | GSA & ESA      | Global   | 23,222                               | 26                  |
| GLONASS       | Roscosmos      | Global   | 19,130                               | 24                  |
| GPS           | US Space Force | Global   | 20,180                               | 31                  |
| IRNSS / NAVIC | ISRO           | Regional | 36,000                               | 8                   |
| QZSS          | JAXA           | Regional | 32,000 (perigree)<br/>40,000(apogee) | 4                   |

## Protocol

Enable and set up the GNSS settings:

| Protocol | Description                                      |
| :------- | :----------------------------------------------- |
| MSP      | GNSS using the MultiWii Serial Protocol          |
| NMEA     | A binary protocol for GNSS data                  |
| UBLOX    | A text protocol for GNSS data, more customizable |

## Settings

| Setting             | Description                                                                                                                                         |
| :------------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------- |
| Auto Baud           | Automatically detect the baud rate of the GNSS                                                                                                      |
| Auto Config         | Automatically configure the GNSS                                                                                                                    |
| Set Home Point Once | If enabled, the home point will be set only once, when a fix is acquired. If disabled, the home point will be updated every time the craft is armed |

For more information about GPS changes in 4.5, see the [Wiki document](/docs/wiki/guides/current/Failsafe).
```